src/_includes/docs/install/devices/ios-physical.md

```diff
@@ -1,3 +1,13 @@
+:::warning
+Flutter's debug mode is broken on physical devices running iOS 18.4 beta.
```

Comment:
WDYT of just using this spot to link to the note in ios-latest.md? Something like:

```
An upcoming change to iOS has caused a temporary break in Flutter's ability to debug code on physical devices running iOS 18.4 (currently in beta). See [Flutter on latest iOS]() for details.

[Flutter on latest iOS]: /platform-integration/ios/ios-latest.md
```